genesis/vis/camera.py

```diff
@@ -160,16 +178,15 @@ def move_to_attach(self):
         ------
         Exception
             If the camera has not been mounted using `attach()`.
-        Exception
-            If the simulation is running in parallel (`n_envs > 0`), which is currently unsupported for mounted cameras.
         """
         if self._attached_link is None:
             gs.raise_exception(f"The camera hasn't been mounted!")
-        if self._visualizer._scene.n_envs > 0:
-            gs.raise_exception(f"Mounted camera not supported in parallel simulation!")
 
-        link_pos = self._attached_link.get_pos().cpu().numpy()
-        link_quat = self._attached_link.get_quat().cpu().numpy()
+        link_pos = self._attached_link.get_pos(envs_idx=self._attached_env_idx).cpu().numpy()
+        link_quat = self._attached_link.get_quat(envs_idx=self._attached_env_idx).cpu().numpy()
+        if self._attached_env_idx is not None:
+            link_pos = link_pos[0] + self._visualizer._scene.envs_offset[self._attached_env_idx]
```

Comment:
why is it `link_pos[0]`?

Reply:
Good question. When querying for `link_pos` from `RigidLink.get_pos()` (through `self._attached_link.get_pos()`), it returns a list of `pos` even when we pass in a single `envs_idx`, which I assume it does, to unify the return type weather we pass in a single `env_idx` or multiple. Since here we always associate a `Camera` with a single environment, hence always have a single `env_idx` (through `self._attached_env_idx`), the expected return would be a list of a _single_ `pos`. So the `link_pos[0]` is to extract that out.

Would you recommend I use `np.squeeze` instead?

Reply:
I see